### DISCUSSION TITLE
Trying to deploy a JS/TS React app for my portfolio via GitHub pages

### DISCUSSION THREAD

**Discussion by mander1** [community/community#49416] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am looking for my first job as a newly trained dev. Obviously a big part of that is to show projects for my portfolio. I have this bug tracker project which I have tried and tried to get up and running using GutHub pages, but no dice, and I've spent days on it; not sure where to go next... Please help! 😭 

repo here https://github.com/mander1/ts-bug-tracker.git

Lewis

**Comment by Anmol-Baranwal** [comment]
> This is not the solution for github pages. 

I would strongly recommend you to deploy your app at [Vercel](https://vercel.com/). 
It's much easier and better for further working on the project.

Vervel will do the job. 

> Another alternative could be [netlify](https://www.netlify.com/). 
<hr>

@mander1 
Check these resources, I think these could resolve your issue of deployment on github-pages:
- https://blog.logrocket.com/deploying-react-apps-github-pages/
- https://github.com/gitname/react-gh-pages

**Comment by mander1** [comment, marked as answer]
Thanks Anmol, appreciate your help.

Lewis

**Reply by Anmol-Baranwal** [reply]
@mander1 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
